misc/systemd/ipfs.service

```diff
@@ -1,9 +1,29 @@
 [Unit]
-Description=IPFS Daemon
+Description=InterPlanetary File System (IPFS) daemon
+After=network.target
 
 [Service]
+
+# enable for 1-1024 port listening
+#AmbientCapabilities=CAP_NET_BIND_SERVICE 
+# enable to specify a custom port
+#Environment=IPFS_PATH=/custom/ipfs/path
+# enable to specify a higher limit for open files/connections
+#LimitNOFILE=1000000
+# enable to specify a higher swarm FD-Limit
+#Environment=LIBP2P_SWARM_FD_LIMIT=800
+# enable to enforce a memory limit
+#MemoryLimit=5000M
+# enable to specify the preferenced connection muxer
+#Environment=LIBP2P_MUX_PREFS="/mplex/6.7.0 /yamux/1.0.0 /spdy/3.1.0"
+# enable to specify a limit for I/O load while starting
+#StartupIOWeight=10
+# enable to specify a limit for I/O load while running
+#IOWeight=50
+
 Type=notify
-ExecStart=/usr/bin/ipfs daemon --init --migrate
+ExecStart=/usr/bin/ipfs daemon --init --migrate --enable-gc --enable-pubsub-experiment --enable-namesys-pubsub
```

Comment:
Definitely don't include these. If users want to enable experiments and/or GC, they can do so themselves.

Reply:
I thought 0.6 was enabling those both (--enable-pubsub-experiment --enable-namesys-pubsub) anyway by default?

And enabling gc by default would follow the principle of least astonishment since there are options in the config file for GC by default.

One would expect that they actually DO anything. But they are completely ignored without adding this parameter.

Reply:
Yes, but that doesn't mean we should be enabling experiments in the default service file.

> And enabling gc by default would follow the principle of least astonishment since there are options in the config file for GC by default.

If we start enabling GC by default, it will be after a thorough audit of GC and likely a rewrite. The systemd unit should not enable things that are not otherwise enabled by default (except `--init` and `--migrate` because those are important for unattended operation).

Reply:
Alright, I revert that :)